### ISSUE TITLE
Incorrect ThreadExceptionDialog

### ISSUE THREAD

**Issue by rido-min** [dotnet/winforms#202]
* .NET Core Version:
```
 Base Path:   C:\Program Files\dotnet\sdk\3.0.100-preview-009812\

Host (useful for support):
  Version: 3.0.0-preview1-27031-02
  Commit:  16fc8e9125
```
* Have you experienced this same bug with .NET Framework? No

**Problem description:**

Thread Exception Dialog shows some issues:

- Resources are not used
- Buttons overlap
- Text is not formatted

![image](https://user-images.githubusercontent.com/14916339/49663771-82702180-fa04-11e8-937a-573b423ea573.png)

**Expected behavior:**
Same as .NET Framework

![image](https://user-images.githubusercontent.com/14916339/49663855-cf53f800-fa04-11e8-81af-d82fe9fb8b3b.png)

**Minimal repro:**
`dotnet new winforms` and edit the Form1.cs to throw an exception from button click

```cs
private void buttonExit_Click(object sender, EventArgs e)
{
    throw new Exception("MyException");
}
```

**Comment by merriemcgaw** [comment]
@rido-min does this still happen on the latest WinForms build? It should be fixed by https://github.com/dotnet/winforms/pull/98

**Comment by rido-min** [comment]
I tried again with latest bits and got the same results:

```
.NET Core SDK (reflecting any global.json):
 Version:   3.0.100-preview-009820
 Commit:    799a33c6ba

Runtime Environment:
 OS Name:     Windows
 OS Version:  10.0.17763
 OS Platform: Windows
 RID:         win10-x64
 Base Path:   C:\Program Files\dotnet\sdk\3.0.100-preview-009820\

Host (useful for support):
  Version: 3.0.0-preview-27205-02
  Commit:  04035b3a4c

.NET Core SDKs installed:
  2.1.600-preview-009426 [C:\Program Files\dotnet\sdk]
  2.2.200-preview-009648 [C:\Program Files\dotnet\sdk]
  3.0.100-preview-009812 [C:\Program Files\dotnet\sdk]
  3.0.100-preview-009820 [C:\Program Files\dotnet\sdk]

.NET Core runtimes installed:
  Microsoft.AspNetCore.All 2.1.6 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.All]
  Microsoft.AspNetCore.All 2.2.0 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.All]
  Microsoft.AspNetCore.App 2.1.6 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 2.2.0 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 3.0.0-preview-18579-0056 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.NETCore.App 2.1.6 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 2.2.0 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 3.0.0-preview-27122-01 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 3.0.0-preview-27205-02 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.WindowsDesktop.App 3.0.0-alpha-27128-4 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]

To install additional .NET Core runtimes or SDKs:
  https://aka.ms/dotnet-download

```

**Comment by rido-min** [comment]
@merriemcgaw I still see this issue with Preview 2, I was expecting that if it was merged it should be fixed in `3.0.100-preview-010184`

**Comment by rido-min** [comment]
@Tanya-Solyanik  In which build can I see the fix?
